### PR TITLE
Finish support for external spamcheck

### PR DESCRIPTION
**Pull request by puiterwijk**

### CODE REVIEW

**Comment by puiterwijk** [comment]
You will most likely want to look at this commit by commit.

File: fas/user.py [hunk 1]
```diff
@@ -1134,6 +1134,32 @@ def accept_user(self, person):
...
+        ''' Accept account from antispam service. '''
+        target = People.by_username(people)
+        user = identity.current.user_name
+
+        # Prevent user from using url directly to update
+        # account if requested's status has been set already.
+        if target.status == status:
+            return {'result': 'nochange'}
+
+        (modo, can_update) = is_modo(user)
```
Comment (ralphbean) [inline fas/user.py]: What does ``modo`` stand for?  Can you add a comment here explaining it?

**Comment by ralphbean** [comment]
Left one cosmetic comment, but the code looks good.  :+1:

File: fas/user.py (hunk 1 above)
Comment (puiterwijk) [inline fas/user.py]: modo is "moderator" in FAS speak.
This is from the current FAS code.

**Comment by puiterwijk** [comment]
Thanks!

**Pull request merged**